aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/input/InputStepTest.java

```diff
@@ -52,11 +54,14 @@ public void parameter() throws Exception {
         InputStepExecution is = a.getExecution("Icecream");
         assertEquals("Do you want chocolate?", is.getInput().getMessage());
         assertEquals(1, is.getInput().getParameters().size());
+        assertEquals("alice", is.getInput().getSubmitter());
 
         j.assertEqualDataBoundBeans(is.getInput().getParameters().get(0), new BooleanParameterDefinition("chocolate", false, "Favorite icecream flavor"));
 
         // submit the input, and run workflow to the completion
-        HtmlPage p = j.createWebClient().getPage(b, a.getUrlName());
+        JenkinsRule.WebClient wc = j.createWebClient();
+        wc.login("alice");
```

Comment:
This will not work unless you have first called

    j.jenkins.setSecurityRealm(j.createDummySecurityRealm());

which you would probably want to do at the top of the test method.